openlibrary/catalog/edition_merge/templates/index.html

```diff
@@ -45,10 +45,11 @@ <h1>Merge duplicate editions</h1>
 </td>
 <td><a href="{{url_for('merge', ia=ia)}}">merge</a></td>
 {% if unmerge_count == 0 %}
-<td style="background: green; color: white">
+<td style="background: green; color: white"></td>
 {% else %}
-<td>
+
 {% endif %}
+<td>
 {{ unmerge_count }}
 </td>
```

Comment:
I'm not sure that this change is needed.  It looks like this is meant to be a single cell that displays the unmerge count.  Do you have any screenshots of this?

Reply:
I am sorry for the late response to this PR, You are right this is not required I will change this and again put a PR @jamesachamp

Reply:
@ArunTeltia any chance of pushing this change? Seems to be the only thing blocking this merge now :)

Reply:
I think we should merge this @jdlrobson sir